include/git2/remote.h

```diff
@@ -688,7 +717,7 @@ typedef struct {
 	git_strarray custom_headers;
 } git_fetch_options;
 
-#define GIT_FETCH_OPTIONS_VERSION 1
+#define GIT_FETCH_OPTIONS_VERSION 2
```

Comment:
We don't update the version number until we have a breaking behavioral change that can be switched by using that.  Adding a new field -- and keeping backward compatibility when it has its default initialization -- won't break consumers, so we don't need to update the version number here.

Reply:
Hmm.. indeed. Just bumping the .so version should be enough, as libraries linking against incompatible ABI won't find their respective .so version anyways